components/member_list_channel/member_list_channel.jsx

```diff
@@ -109,6 +109,11 @@ export default class MemberListChannel extends React.PureComponent {
     }
 
     render() {
+        if (this.state.loading) {
+            return (<div >
```

Comment:
I would use a `<React.Fragment>` here instead of a `<div>`. But, is the wrapper `<div>` really necessary?

Reply:
No, you're absolutely correct. Removed the div 👍